api/src/main/java/org/openmrs/module/pihcore/metadata/Metadata.java

```diff
@@ -79,4 +88,44 @@ public static Concept getConcept(String lookup) {
         }
         return c;
     }
+
+    /**
+     * TODO: grant GET_LOCATIONS privilege to Anonymous instead of using a proxy privilege
+     */
```

Comment:
@mogoodrich do you think we should just go ahead and finally do this?  It seems right to me.  Then I can remove the proxy privilege stuff from here.

Reply:
Seems reasonable?  I don't have a strong preference.